spk/synocli-devel/Makefile

```diff
@@ -90,25 +88,27 @@ SPK_COMMANDS += bin/strip
 ##
 ## gdb
 ##
-ifeq ($(findstring $(ARCH),$(ARMv5_ARCHS)),$(ARCH))
+ifeq ($(findstring $(ARCH),$(ARMv5_ARCHS) $(PPC_ARCHS)),$(ARCH))
+OPTIONAL_DESC += ", gdb 7.12.1"
+else
+ifeq ($(call version_lt, $(TC_GCC), 4.8),1)
 OPTIONAL_DESC += ", gdb 7.12.1"
 else
 OPTIONAL_DESC += ", gdb 13.2"
 endif
+endif
 SPK_COMMANDS += bin/gcore
 SPK_COMMANDS += bin/gdb
 SPK_COMMANDS += bin/gdb-add-index
 SPK_COMMANDS += bin/gdbserver
-ifeq ($(findstring $(ARCH),$(ARMv5_ARCHS) $(ARMv7_ARCHS) $(ARMv7L_ARCHS)),$(ARCH))
+ifeq ($(findstring $(ARCH),$(ARM_ARCHS)),$(ARCH))
 SPK_COMMANDS += bin/run
```

Comment:
This would also include aarch64 no?  But I believe it shouldn't.  To be confirmed.

Reply:
Yes it does. `bin/run` is also built for aarch64. (if not, the build would fail).